bindings/py/cpp_src/bindings/engine/py_OS.cpp

```diff
@@ -1,45 +0,0 @@
-/* ---------------------------------------------------------------------
- * HTM Community Edition of NuPIC
- * Copyright (C) 2018, Numenta, Inc.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero Public License version 3 as
- * published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
- * See the GNU Affero Public License for more details.
- *
- * You should have received a copy of the GNU Affero Public License
- * along with this program.  If not, see http://www.gnu.org/licenses.
- *
- * Author: @chhenning, 2018
- * --------------------------------------------------------------------- */
-
-/** @file
-PyBind11 bindings for OS classes
-*/
-
-
-#include <bindings/suppress_register.hpp>  //include before pybind11.h
-#include <pybind11/pybind11.h>
-#include <pybind11/iostream.h>
-#include <pybind11/numpy.h>
-#include <pybind11/stl.h>
-
-#include <htm/os/OS.hpp>
-
-namespace py = pybind11;
-
-using namespace htm;
-
-namespace htm_ext {
-
-    void init_OS(py::module& m)
-    {
-        py::class_<OS> py_OS(m, "OS");
-        py_OS.def_static("getProcessMemoryUsage", &htm::OS::getProcessMemoryUsage);
```

Comment:
this is the only function that could have been somewhat useful, but even that is never used.

Reply:
Python already has this in the standard library, albeit for Unix only:
https://docs.python.org/3.6/library/resource.html

Reply:
Cool, I thought I've seen it used in your param-optimizer in python. 
Our "custom" code worked only on UNIX too, so no problem getting rid of it.